numpy/_core/tests/test_ufunc.py

```diff
@@ -443,7 +443,7 @@ def test_signature_failure_signature_missing_output_arg(self):
             umt.test_signature(2, 2, "(i),(i)->()")
 
     def test_get_signature(self):
-        assert_equal(umt.inner1d.signature, "(i),(i)->()")
+        assert_equal(np.vecdot.signature, "(n),(n)->()")
```

Comment:
These changes of `umt.inner1d` to `np.vecdot` are not directly related to the PR, but I thought one might as well rely on an actual gufunc, rather than one created for testing purposes only.